docs/stylesheets/extra.css

```diff
@@ -25,3 +25,76 @@ based on this suggestion: https://github.com/squidfunk/mkdocs-material/discussio
   padding-top: .175rem;
   padding-bottom: .175rem;
 }
+
+/*
+Define Imageomics colors for theme (light and dark)
+Primary Colors:
+- Imageomics Green: #92991c
+- Imageomics Blue: #5d8095
+- Dark Teal: #0097b2
+- White: #FFFFFF
+Accent Colors:
+- Light Green: #9bcb5e
+- Red: #bb0000
+- NSF Gold: #caae54
+- NSF Blue: #61a5d6
+Adjust admonitions and links to match (where otherwise clashed)
+*/
+
+/* Light mode colors: Imageomics Blue and Light Green */
+[data-md-color-scheme="default"] {
+  --md-primary-fg-color:        #5d8095;
+  --md-accent-fg-color:         #9bcb5e;
+}
+
+/* Set admonition (Note) colors to match Dark Teal used for URLs */
+/* border color */
+.md-typeset .admonition.note, .md-typeset details.note {
+    border-color: #0097b2;
+    box-shadow: #0097b21a;
+}
+
+/* icon color */
+.md-typeset .admonition.note > .admonition-title::before {
+    background-color: #0097b2;
+}
+
+/* shaded part (title/heading) */
+.md-typeset .note>.admonition-title,.md-typeset .note>summary {
+    background-color: #0097b21a;
```

Comment:
Interesting, I learned something new today about the alpha channel. I don't think this used to be there originally?